### DISCUSSION TITLE
Add an $attributes property to a component for renaming of fields in validation messages

### DISCUSSION THREAD

**Discussion by allmanaj** [livewire/livewire#1637] [4. General]
This would work in much the same way as `$messages` or `$rules` but would allow the user to define custom names for their fields to be shown in validation messages.

For example, if the user has a form component and wants to keep all values in a `$form` property as an associative array then this will enable them to avoid having error messages like `the form.name field is required`.

e.g
```php
$attributes = [
    'form.name' => 'name'
];
```
would change the error message from `the form.name field is required` to `the name field is required`

I currently have this change in my local version of livewire so can create a PR quickly, but I wanted to create this issue first so that it could be discussed.

Thanks for all the amazing work 🙌

**Comment by zepfietje** [comment]
@allmanaj, https://github.com/livewire/livewire/pull/1643 is probably what you want.
